README.md

```diff
@@ -10,6 +10,9 @@
   <a href="https://github.com/web3-storage/w3ui/blob/main/LICENSE.md"><img alt="License: Apache-2.0 OR MIT" src="https://img.shields.io/badge/LICENSE-Apache--2.0%20OR%20MIT-yellow?style=for-the-badge" /></a>
 </p>
 
+# 📣❗ w3ui and the underlying APIs are currently beta preview features
+Please read the beta [Terms of Service](https://purrfect-tracker-45c.notion.site/w3up-beta-Terms-of-Service-39cb5c13439849beae327a2efec9164a) for more details. Open an issue on the repo or reach out to the #web3-storage channel on [IPFS Discord](https://docs.ipfs.tech/community/chat/#discord) if you have any questions!
```

Comment:
How about this:

```suggestion
> ### ⚠️❗ w3ui and the underlying APIs are currently BETA preview features
> Please read the beta [Terms of Service](https://purrfect-tracker-45c.notion.site/w3up-beta-Terms-of-Service-39cb5c13439849beae327a2efec9164a) for more details.
>
> Open an issue on the repo or reach out to the #web3-storage channel on [IPFS Discord](https://docs.ipfs.tech/community/chat/#discord) if you have any 
questions!
```

Renders as:

<img width="1197" alt="Screenshot 2022-12-15 at 17 00 45" src="https://user-images.githubusercontent.com/152863/207922344-f54b6b11-f03e-45b0-a257-d37d477c0175.png">

VS

<img width="1231" alt="Screenshot 2022-12-15 at 17 01 50" src="https://user-images.githubusercontent.com/152863/207922441-394a00be-fe89-4d59-927e-8c1eacedf303.png">